.github/workflows/ci.yml

```diff
@@ -73,9 +73,9 @@ jobs:
         run: |
           $TRAVIS_DIR/run-unit-test.sh $BACKEND
 
-      - name: Run core test
-        run: |
-          $TRAVIS_DIR/run-core-test.sh $BACKEND
+#      - name: Run core test
+#        run: |
+#          $TRAVIS_DIR/run-core-test.sh $BACKEND
```

Comment:
The core tests for the hstore backend in `pd-store.yml` should be commented out, not the core tests for the original backend~